### ISSUE TITLE
Invalid mount config path does not exist `.../configs/flags.yml`

### ISSUE THREAD

**Issue by marcosmarxm** [airbytehq/airbyte#22144]
## Environment
- **Airbyte version**:  0.40.32
- **OS Version / Instance**: macOS
- **Deployment**: Docker version 20.10.22, build 3a2c30b
- **Source Connector and version**: NA
- **Destination Connector and version**: NA
- **Step where error happened**: **Deploy** / Sync job / Setup new connection / Update connector / Upgrade Airbyte

## Current Behavior

Our current documentation to deploy Airbyte in Cloud Providers and to upgrade previous instances follow the instructions:
```
mkdir airbyte && cd airbyte
curl -sOO https://raw.githubusercontent.com/airbytehq/airbyte/master/{.env,docker-compose.yaml}
docker compose up -d
```
See example: https://docs.airbyte.com/deploying-airbyte/on-gcp-compute-engine#install-and-launch-airbyte

The PR https://github.com/airbytehq/airbyte/pull/21569 introduced a dependency of file `config/flags.yml` which is not included in our docs.
Some users are reporting the problem when trying to deploy new instances or upgrade old ones.

Logs:
```
➜  test docker compose up -d
WARN[0000] The "NEW_SCHEDULER" variable is not set. Defaulting to a blank string.
WARN[0000] The "SECRET_PERSISTENCE" variable is not set. Defaulting to a blank string.
WARN[0000] The "JOB_ERROR_REPORTING_SENTRY_DSN" variable is not set. Defaulting to a blank string.
WARN[0000] The "WORKER_ENVIRONMENT" variable is not set. Defaulting to a blank string.
WARN[0000] The "GITHUB_STORE_BRANCH" variable is not set. Defaulting to a blank string.
WARN[0000] The "DEPLOYMENT_MODE" variable is not set. Defaulting to a blank string.
WARN[0000] The "LOG_CONNECTOR_MESSAGES" variable is not set. Defaulting to a blank string.
WARN[0000] The "SHOULD_RUN_NOTIFY_WORKFLOW" variable is not set. Defaulting to a blank string.
WARN[0000] The "SECRET_PERSISTENCE" variable is not set. Defaulting to a blank string.
WARN[0000] The "JOB_ERROR_REPORTING_SENTRY_DSN" variable is not set. Defaulting to a blank string.
WARN[0000] The "APPLY_FIELD_SELECTION" variable is not set. Defaulting to a blank string.
WARN[0000] The "FIELD_SELECTION_WORKSPACES" variable is not set. Defaulting to a blank string.
WARN[0000] The "DEPLOYMENT_MODE" variable is not set. Defaulting to a blank string.
WARN[0000] The "REMOTE_CONNECTOR_CATALOG_URL" variable is not set. Defaulting to a blank string.
WARN[0000] The "TEMPORAL_HISTORY_RETENTION_IN_DAYS" variable is not set. Defaulting to a blank string.
WARN[0000] The "UPDATE_DEFINITIONS_CRON_ENABLED" variable is not set. Defaulting to a blank string.
[+] Running 10/10
 ⠿ Network test_airbyte_internal               Created                     0.0s
 ⠿ Network test_airbyte_public                 Created                     0.0s
 ⠿ Network test_default                        C...                        0.0s
 ⠿ Volume "airbyte_workspace"                  Created                     0.0s
 ⠿ Volume "airbyte_db"                         Cr...                       0.0s
 ⠿ Volume "airbyte_data"                       Created                     0.0s
 ⠿ Container init                              Created                     0.1s
 ⠿ Container airbyte-temporal                  Created                     0.2s
 ⠿ Container airbyte-db                        C...                        0.1s
 ⠿ Container airbyte-bootloader                Created                     0.1s
 ⠋ Container airbyte-connector-builder-server  Creating                    0.0s
 ⠋ Container airbyte-cron                      Creating                    0.0s
 ⠋ Container airbyte-server                    Creating                    0.0s
 ⠋ Container airbyte-worker                    Creating                    0.0s
 ⠋ Container airbyte-webapp                    Creating                    0.0s
Error response from daemon: invalid mount config for type "bind": bind source path does not exist: /host_mnt/Users/marcosmarx/test/configs/flags.yml
```

## Expected Behavior
Our docs should give instructions without problems.

## Steps to Reproduce
1. Follow instructions (one without pulling the entire project)
2. Have Docker V2 installed
3. `docker compose up -d`

## Are you willing to submit a PR?

No

**Comment by marcosmarxm** [comment]
@colesnodgrass fyi

**Comment by marcosmarxm** [comment]
The recommendation to fix the issue is to wget/curl the file manually.

**Comment by colesnodgrass** [comment]
@marcosmarxm this should be fixed by #22158 and the documentation was updated in #22160 to reflect this new file

**Comment by dan-r95** [comment]
For me, setting shared folders in docker-desktop was necessary
https://github.com/airbytehq/airbyte/issues/33845#issuecomment-2133715252
